azkaban/__main__.py

```diff
@@ -420,30 +426,54 @@ def run_workflow(project_name, _flow, _job, _url, _alias, _bounce, _kill,
   )
 
 def schedule_workflow(project_name, _date, _time, _span, _flow, _job, _url,
-  _alias, _bounce, _kill, _email, _option, _jump, _notify_early, _mode):
-  """Schedule workflow."""
-  session = _get_session(_url, _alias)
-  kwargs = {
-    'name': project_name,
-    'flow': _flow,
-    'date': _date,
-    'time': _time,
-    'period': _span,
-    'concurrent': _mode if _mode else not _bounce,
-    'on_failure': 'cancel' if _kill else 'finish',
-    'emails': _email,
-    'properties': _parse_option(_option),
-  }
-  if _notify_early:
-    kwargs['notify_early'] = True
-  if _jump:
-    kwargs['disabled_jobs'] = _job
+  _alias, _bounce, _kill, _email, _option, _jump, _notify_early, _mode, _cron_exp, _timezone):
+  if _cron_exp:
+    """Schedule cron workflow."""
```

Comment:
This function is missing a docstring, and triple quote strings inside functions are highly unusual. Can you add the docstring back and remove these internal ones?